plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -387,7 +387,7 @@ export function wrapFieldResolver<TSource = any, TContext = any, TArgs = any>(
             >
         >(
           () => {
-            return fieldResolver.call(this, source, args, contextValue, info);
+            return fieldResolver.call(this, source, args, contextValue, info) as any;
```

Comment:
The `any` used to be baked into the generic return type, whereas it now uses `unknown` instead, which is insufficient here.